### PR TITLE
make the pow description more clear with right parameter names

### PR DESCRIPTION
**Pull request by dylanvu**
image

### CODE REVIEW

File: project_3.html [hunk 1]
```diff
@@ -114,167 +114,209 @@ <h5 class="mt-2 mb-2 mx-3">Contents</h5>
...
+						Function Header: <code class="pj-code language-arduino">bool isEven(int num)</code>
+					</li>
+					<li class="list-group-item">
+						Parameters: num (int)
+					</li>
+					<li class="list-group-item">
+						Return: bool
+					</li>
+					<li class="list-group-item">
+						Description: <code class="pj-code language-arduino">isEven()</code> returns true if the
```
Comment (bcbielecki) [inline project_3.html:265]: Remove the "()" from "isEven()"

File: project_3.html [hunk 2]
```diff
@@ -283,175 +325,210 @@ <h3 class="mb-3" id="pj-heading-take-the-average">Take the Average</h3>
 							Return: double
 						</li>
 						<li class="list-group-item">
-							Description: <code class="pj-code language-arduino">avg()</code> returns the mean of the elements in the array <code class="pj-code language-arduino">arr</code>. The array must not be mutated by the function. Its elements may be positive or negative.
+							Description: <code class="pj-code language-arduino">avg()</code> returns the mean of the
```
Comment (bcbielecki) [inline project_3.html:328]: Remove "()" from "avg()"

File: project_3.html [hunk 3]
```diff
@@ -283,175 +325,210 @@ <h3 class="mb-3" id="pj-heading-take-the-average">Take the Average</h3>
...
+						Function Header: <code class="pj-code language-arduino">void fib(unsigned n)</code>
+					</li>
+					<li class="list-group-item">
+						Parameters: n (unsigned int)
+					</li>
+					<li class="list-group-item">
+						Return: None
+					</li>
+					<li class="list-group-item">
+						Description: <code class="pj-code language-arduino">fib()</code> prints the first n elements of
```
Comment (bcbielecki) [inline project_3.html:347]: Remove "()" from "fib()"

Comment (dylanvu) [reply project_3.html:347]: done

File: project_3.html (hunk 2 above)
Comment (dylanvu) [reply project_3.html:328]: done

File: project_3.html [hunk 4]
```diff
@@ -114,167 +114,209 @@ <h5 class="mt-2 mb-2 mx-3">Contents</h5>
...
+							class="pj-code language-arduino">unsigned pow2(unsigned base, unsigned exp)</code>
+					</li>
+					<li class="list-group-item">
+						Parameters: base (unsigned int), exp (unsigned int)
+					</li>
+					<li class="list-group-item">
+						Return: unsigned int
+					</li>
+					<li class="list-group-item">
+						Description: <code class="pj-code language-arduino">pow2(base, exp)</code> returns the
```
[the remark this answers is not in the document]
Comment (dylanvu) [reply project_3.html:296]: done

File: project_3.html (hunk 1 above)
Comment (dylanvu) [reply project_3.html:265]: done

File: project_3.html [hunk 5]
```diff
@@ -114,167 +114,209 @@ <h5 class="mt-2 mb-2 mx-3">Contents</h5>
...
+						Function Header: <code class="pj-code language-arduino">void echo()</code>
+					</li>
+					<li class="list-group-item">
+						Parameters: None
+					</li>
+					<li class="list-group-item">
+						Return: None
+					</li>
+					<li class="list-group-item">
+						Description: <code class="pj-code language-arduino">echo()</code> uses the standard input/output
```
[the remark this answers is not in the document]
Comment (dylanvu) [reply project_3.html:246]: done

**Review by bcbielecki — APPROVED** [review]

**Pull request merged**